package.json

```diff
@@ -65,6 +65,7 @@
     "test": "nyc mocha 'test/**/*.test.{ts,tsx}'",
     "test-watch": "mocha 'test/**/*.test.{ts,tsx}' -w",
     "prepare": "husky install",
+    "start-example": "rm -rf node_modules/@types/react node_modules/react && cd example && yarn start",
```

Comment:
Not needed here imo. A cleaner example on how to fix this is explained here: https://github.com/arcxmoney/analytics-sdk/tree/feat/signing-track/example#note